riscv-target/opentitan/device/rv32imc/Makefile.include

```diff
@@ -0,0 +1,76 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+
+OT           = $(ROOTDIR)/riscv-target/$(RISCV_TARGET)/device/rv32imc
+OTSW         = $(ROOTDIR)/../../device
+OTROOT       = $(OTSW)/../../
+LDSCRIPT     = $(OT)/link.ld
+TRAPHANDLER  = $(OT)/handler.S
+DEFINES      = $(CARG) -DPRIV_MISA_S=0 -DPRIV_MISA_U=0 -DTRAPHANDLER="\"$(TRAPHANDLER)\""
+RV_TOOLS    ?= /tools/riscv/bin
+FPGA_UART   ?=
+TARGET_SIM  ?= $(OTROOT)/build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator
+TARGET      ?= fpga
```

Comment:
Can you use variable names which make it very clear which ones are opentitan-specific and which ones are the ones commonly used in the riscv-compliance test suite? E.g. prefix all opentitan variables with `OT_` or `OPENTITAN_`.

Reply:
fixed, at least IMO. Let me know if you prefer other names